app/bundles/FormBundle/Translations/en_US/messages.ini

```diff
@@ -136,7 +136,7 @@ mautic.form.form.help.searchcommands="<strong>Search commands</strong><br />ids:
 mautic.form.form.kioskmode="Kiosk Mode"
 mautic.form.form.kioskmode.tooltip="If set to yes, form submissions will not generate new contact tracking cookies or assign the IP address to created/updated contacts."
 mautic.form.form.renderstyle="Render Style"
-mautic.form.form.renderstyle.tooltip="Disable CSS rendering"
+mautic.form.form.renderstyle.tooltip="Disable/Enable rendering of CSS"
```

Comment:
```suggestion
mautic.form.form.renderstyle.tooltip="Disable/Enable rendering of CSS from Template"
```

Reply:
I added the part "from template", but leaved the order "Disable/Enable" as on the left we have "No" and on the right we have "Enable", so "Disable/Enable" follows the order of the choices.